src/services/VoteService.ts

```diff
@@ -4,70 +4,62 @@ import VoteRepository from "../repositories/VoteRepository";
 import { ObjectId } from "mongodb";
 
 export interface VoteResult {
-  questionIdentifier: string,
-  questionLabel: string,
-  amount: number
+  questionIdentifier: string;
+  questionLabel: string;
+  amount: number;
 }
 
 export interface TotalVoteCount {
-  count: number
+  count: number;
 }
 
 export class VoteService {
   public async saveVote(ballotID: string, token: string, vote: string) {
-    await VoteRepository.addVote(
-      token,
-      (await BallotRepository.getBallot(ballotID))._id,
-      vote,
-      new Date()
-    );
+    const ballot = await BallotRepository.getBallot(ballotID);
+    await VoteRepository.addVote(token, ballot._id, vote, new Date());
   }
 
   public async getVote(ballotID: string, token: string): Promise<Vote> {
     return await VoteRepository.getVote(token, new ObjectId(ballotID));
   }
 
   public async getVotes(ballotID: string): Promise<Vote[]> {
-    return await VoteRepository.getVotes(new ObjectId(ballotID))
+    return await VoteRepository.getVotes(new ObjectId(ballotID));
   }
 
   public async checkIfAlreadyVoted(
     ballotID: string,
     token: string
   ): Promise<boolean> {
-    return (await this.getVote(ballotID, token)) != null;
+    const vote = await this.getVote(ballotID, token);
+    return vote != null;
```

Comment:
meh why different from 
https://github.com/LeoTuet/backend-we-want-to-sleep/blob/bce08ca414ba9b9a0351675d3196a42a5a3a2c23/src/services/VoteService.ts#L23
or
https://github.com/LeoTuet/backend-we-want-to-sleep/blob/bce08ca414ba9b9a0351675d3196a42a5a3a2c23/src/services/VoteService.ts#L27

Reply:
Bc there is no != null